generators/service-ibm-alert-notification/templates/localdev-config.json.template

```diff
@@ -0,0 +1,6 @@
+{
+  "ibm-alert-notification_name": "{{name}}",
```

Comment:
let's just use alert-notification, no "ibm". also, please use snake_case, no dashes.

Reply:
Why would application need swaggerui? How name, password and url properties are used?